tests/core/fixtures.py

```diff
@@ -76,6 +83,29 @@ def chain():
     "0xf90260f901f9a07285abd5b24742f184ad676e31f6054663b3529bc35ea2fcad8a3e0f642a46f7a01dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347948888f1f195afa192cfee860698584c030f4c9db1a0964e6c9995e7e3757e934391b4f16b50c20409ee4eb9abd4c4617cb805449b9aa053d5b71a8fbb9590de82d69dfa4ac31923b0c8afce0d30d0d8d1e931f25030dca0bc37d79753ad738a6dac4921e57392f145d8887476de3f783dfa7edae9283e52b90100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000008302000001832fefd8825208845754132380a0194605bacef646779359318c7b5899559a5bf4074bbe2cfb7e1b83b1504182dd88e0205813b22e5a9cf861f85f800a82c35094095e7baea6a6c7c4c2dfeb977efac326af552d870a801ba0f3266921c93d600c43f6fa4724b7abae079b35b9e95df592f95f9f3445e94c88a012f977552ebdb7a492cf35f3106df16ccb4576ebad4113056ee1f52cbe4978c1c0")  # noqa: E501
 
 
+def import_block_without_validation(chain, block):
+    """
+    A copy of `Chain.import_block` without the block validation.
+    """
+    if block.number > chain.header.block_number:
+        raise ValidationError(
+            "Attempt to import block #{0}.  Cannot import block with number "
+            "greater than current block #{1}.".format(
+                block.number,
+                chain.header.block_number,
+            )
+        )
+
+    parent_chain = chain.get_parent_chain(block)
+    imported_block = parent_chain.get_vm().import_block(block)
+
+    persist_block_to_db(chain.db, imported_block)
+    if chain.should_be_canonical_chain_head(imported_block):
+        chain.add_to_canonical_chain_head(imported_block)
+
+    return imported_block
```

Comment:
I'm a bit worried about duplicating the whole import_block() method here as there's nothing to ensure both implementations are identical. It could lead to tests that pass just because we're not exercising the actual production code. That's not an issue when we simply make the validation method a no-op

Reply:
Updated to use the kwarg approach we talked about.